components/style/sharing/mod.rs

```diff
@@ -584,14 +525,7 @@ impl<E: TElement> StyleSharingCandidateCache<E> {
 
             match sharing_result {
                 Ok(shared_styles) => {
-                    // Yay, cache hit. Share the style.
-                    let child_cascade_requirement =
-                        target.accumulate_damage_when_sharing(shared_context,
```

Comment:
Can you explain for me when we no longer need to do this?  (I assume it's already done elsewhere.)